docs/Best Practice Documents/1.2. SDL App Icon Display/index.md

```diff
@@ -0,0 +1,117 @@
+# 1.2. SDL App Icon Display
```

Comment:
I believe most of the information is covered by the app icon resumption feature. If the questions could be answered for this section that will better help us decide where certain parts of this guide should be included in existing documentation.

Reply:
The Proposal of SDL-0041 is to use the Icon that was successfully displayed next time, or to use the default Icon if it fails.
This chapter describes how to display Icon according to various conditions between App and HU. This chapter provides reference information centered on the HMI.
We want SDLC members to choice the contents of the required documents and apply them to existing documents.